### ISSUE TITLE
@at-root freezes lint if used as inline selector

### ISSUE THREAD

**Issue by alexerlandsson** [sasstools/sass-lint#1014]
When using @at-root as a inline selector the lint freezes and won't be able to finish. It started to work when I changed the @at-root rule to wrap the selector instead.

No errors was printed in the Terminal and the lint did not finish so I had to try the lint to each separate file before I found the problem.

This example won't work:
```scss
@at-root .element {
  color: red;
}
```

This will work:
```scss
@at-root {
  .element {
    color: red;
  }
}
```

Reading the scss documentation, @at-root should work both to inline selectors and groups (http://www.sass-lang.com/documentation/file.SASS_REFERENCE.html#at-root).

Sass Lint version: 1.10.2

**Comment by DanPurdy** [comment]
@alexerlandsson this is a bug in the current version of the AST we use. It looks to be fixed in V4 but we've not had the time to update to this yet or even check all the possible regressions.

**Comment by bgriffith** [comment]
This should be fixed in the newly released version v1.11.0. Please let me know if you still have issues. Thanks!
